humility-core/Cargo.toml

```diff
@@ -41,7 +41,7 @@ hex = "0.4.3"
 # We depend on the oxide-stable branch of Oxide's fork of probe-rs to assure
 # that we can float necessary patches on probe-rs.
 #
-probe-rs = { git = "https://github.com/oxidecomputer/probe-rs.git", branch = "oxide-v0.12.0" }
+probe-rs = { git = "https://github.com/rivosinc/probe-rs.git", branch = "dev/zheren/humility_bump" }
```

Comment:
Also, this branch looks old, last commit was in July.  We need to at least include `[f27c9c856f1972d6482e30559ff7176aad70f751](https://github.com/probe-rs/probe-rs/commit/f27c9c856f1972d6482e30559ff7176aad70f751)` as it has some CSR bug fixes.

Reply:
> Whats different about your branch compared to upstream?

When you say "upstream," do you mean official probe-rs or oxide's fork? My branch uses probe-rs' [v0.13.0](https://github.com/probe-rs/probe-rs/tree/v0.13.0) release. Note that they (maybe by mistake, they did have one for v0.12.0) didn't create a branch for v0.13.0 release, instead they only have a tag. So I created my branch using that tag to get v0.13.0.

Plus, they haven't released v0.14.0. I kind of feel like it's better to keep the latest released version instead of keeping up-to-date with the master branch. So If we need it, maybe we can cherry-pick that commit?

Reply:
>When you say "upstream," do you mean official probe-rs or oxide's fork?

I was referring to the official [probe-rs](https://github.com/probe-rs/probe-rs).

Reply:
I would think it is fine to just point to master in this case, I just don't see a benefit of forking it in this case.  The hash revision is pinned, so its not like we have to worry about the active development changing things for us.  It would save us the hassle of having to maintain a fork and back-porting any relevant patches.  Might defer to some of the more experienced folks to chime in though. (@furquan-rivos, @richard-rivos, etc )

Reply:
What @beezow suggested makes sense to me. I think rather than creating another fork, we can use the latest official probe-rs directly. If we expect to have many local changes to carry in probe-rs, we can also fork and create a rivosinc repo, but it might be better to use rivos/main rather than a development branch for pinning here.

Once we upstream all the humility changes to oxide, we probably would want to converge on a common probe-rs.

Reply:
Ack. Will change it to use latest upstream probe-rs master branch.

Reply:
> Will change it to use latest upstream probe-rs master branch.

Done.